src/components/view/pages/payment/paymentPageView.ts

```diff
@@ -221,13 +229,11 @@ export class PaymentPageView extends View<CartDataType> {
             cvvInput.value = cvvInput.value.replace(/[^\0-9]/g, '').substring(0, 3);
 
             if (cvvInput.value.match(/^\d{3}/g)) {
-                cvvInput.classList.add('valid');
-                cvvInput.classList.remove('invalid');
-                validCardCvv = true;
+                valid.validCardCvv = true;
+                setInputValidity(cvvInput, valid.validCardCvv);
             } else {
-                cvvInput.classList.remove('valid');
-                cvvInput.classList.add('invalid');
-                validCardCvv = false;
+                valid.validCardCvv = false;
+                setInputValidity(cvvInput, valid.validCardCvv);
             }
         })
```

Comment:
Please try to use func for all your inputs check
`const checkCvvInput = (value) => value.match(/^\d{3}/g)`

And your code will be:

`const isValidCvvInput = checkCvvInput(cvvInput.value)

valid.validCardCvv = isValidCvvInput

if (isValidCvvInput) {
makeInputValid()
} else {
makeInputInvalid()
}`